src/EventHandlers/Voter.ts

```diff
@@ -59,9 +59,9 @@ Voter.GaugeCreated.handler(async ({ event, context }) => {
   // The pool entity should be created via PoolCreated event from the PoolFactory contract
   // Store pool details in poolRewardAddressStore
   let currentPoolRewardAddressMapping = {
-    poolAddress: event.params.pool,
-    gaugeAddress: event.params.gauge,
-    bribeVotingRewardAddress: event.params.bribeVotingReward,
+    poolAddress: toChecksumAddress(event.params.pool),
+    gaugeAddress: toChecksumAddress(event.params.gauge),
+    bribeVotingRewardAddress: toChecksumAddress(event.params.bribeVotingReward),
```

Comment:
Wondering the computational cost of all this checksumming, if we've removed the toLowerCase we shouldn't need this now right?